confs/scaleway-03/sanoid/sanoid.conf

```
@@ -0,0 +1,118 @@
+# sanoid conf data - see https://github.com/jimsalterjrs/sanoid/wiki/Sanoid
+
+
+# VM and CT volumes stored on HDD disks
+[zfs-hdd/pve]
+  use_template=local_sys
+  recursive=yes
+
+[zfs-hdd/backups]
+  use_template=local_data
+  recursive=yes
```

if you have data synced from other location you should not tell sanoid to create the snapshot, it should only handle retention, the snapshots are done on the source.
So you have to use 
```suggestion
[zfs-hdd/backups]
  use_template=synced_data
  recursive=yes
```
where we have `frequently = 0`

---

👍🏻